bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -20,20 +20,40 @@ Binding for the Bosch Smart Home Controller:
 
 ## Discovery
 
-Not yet implemented. Configuration via configuration files or Paper UI.
+Not yet implemented. Configuration via configuration files or Paper UI (see below).
 
 ## Binding Configuration
 
-You need to provide at least the ipAddress and the system password of your Bosch Smart Home Controller
-and a path to a JKS keystore file location.
-At this location a keystore file with a self signed certificate is created and stored if not present at the first setup.
-This certificate is used for pairing between the Bridge and the Bosch SHC. 
+You need to provide at least the IP address and the system password of your Bosch Smart Home Controller. The IP address of the controller is visible in the Bosch Smart Home Mobile App (More -> System -> Smart Home Controller) or in your network router UI.
```

Comment:
Do we need to document more, because the setup wasn't clear, see #22 ?

Reply:
@GerdZanker Totally missed that issue #22, but I'm glad that you got it working for them. Although I didn't fully understand what went wrong for them in the end. Feel free to add any information that could help to that paragraph. 

In my opinion it was not necessary anymore to mention the stuff about the certificates because they are created automatically. So that might just add more confusion than it might help. But adding some more explanation e.g. what "system password" means, could help!

Reply:
> it was not necessary anymore to mention the stuff about the certificates 
Correct, that's not necessary

Reply:
I took a look into the [Englisch SHC manual (Page 11)](https://tt-smarthome.resource.bosch.com/NRT/Web/Produkt/Controller/PDF/SHC_manual_EN_04_2019.pdf) and the term "system password" is used here. Therefore I would refer to this registration step.